integrations/lib/multiwoven/integrations/destination/zendesk/config/catalog.json

```diff
@@ -0,0 +1,110 @@
+{
+    "request_rate_limit": 700,
+    "request_rate_limit_unit": "minute",
+    "request_rate_concurrency": 10,
+    "streams": [
+        {
+            "name": "Tickets",
```

Comment:
We usually use the same name of the API path i.e `tickets` in https://developer.zendesk.com/api-reference/ticketing/introduction/ or function name in the gem which is also `tickets` and this is sent to the function inside the client via frontend. So we can avoid using pluralization etc